service/errors.go

```diff
@@ -7,31 +7,31 @@ import (
 var (
 	// Errors lists all available error types
 	Errors = []*types.Error{
-		errNotReady,
-		errNotImplemented,
-		errNotSupported,
-		errUnavailableOffline,
-		errInternalError,
-		errInvalidInput,
-		errClientError,
-		errBlockInvalidInput,
-		errBlockNotFound,
-		errCallInvalidMethod,
-		errCallInvalidParams,
+		ErrNotReady,
+		ErrNotImplemented,
+		ErrNotSupported,
+		ErrUnavailableOffline,
+		ErrInternalError,
+		ErrInvalidInput,
+		ErrClientError,
+		ErrBlockInvalidInput,
+		ErrBlockNotFound,
+		ErrCallInvalidMethod,
+		ErrCallInvalidParams,
```

Comment:
We are making the errors and WrapError public in order to access them from the backend packages. The dependency goes from Backend -> Service. We inject the concrete backends via service constructors.